src/Connector.php

```diff
@@ -113,7 +133,10 @@ private function postClientRequest(string $method, string $apiPath, array $optio
         $this->logger->debug("Returned response {$statusCode}", $content);
 
         if ($statusCode !== 200) {
-            throw new ApiException("Unexpected response status code '{$statusCode}'", $statusCode);
+            $message = "Unexpected response status code '{$statusCode}'";
+            $statusMessage = $content["error"] ?? $content['message'] ?? '';
```

Comment:
error written with double quotes, message with single quotes, I guess we should use single quotes